gui/src/components/AppSidebar.tsx

```diff
@@ -201,5 +201,6 @@ const settingsItems = [
   { title: "Network", url: "/home/settings/networks" },
   { title: "Foundry", url: "/home/settings/foundry" },
   { title: "Tokens", url: "/home/settings/tokens" },
+  { title: "Stacks", url: "/home/settings/stacks" },
```

Comment:
missing the logic to hide this menu option if the flag is not enabled
(I think hiding the link should be enough. you can technically still navigate to it via the console, but that's fine)

Reply:
On the Rust side, we have the feature to know if the feature is enabled. Is there currently a way for the TypeScript side to detect whether this flag is active? If not, should we introduce a global setting value that indicates the feature is enabled ?